read-fonts/src/tables/instance_record.rs

```diff
@@ -0,0 +1,70 @@
+//! An fvar InstanceRecord
+
+use super::InstanceRecord;
```

Comment:
any reason not to define the type in this file?

Reply:
Oh, I think I had done everything in `fvar.rs` before I noticed the pattern you used with `ValueRecord` and just didn't copy the base type over. I'll update it.

Reply:
Turns out the fields on that type weren't pub either, so I'm glad you called this out :)